script/upgrades/MU01-mcmint/governance/MU01-Baklava-CGP.simulation.sol

```diff
@@ -4,16 +4,15 @@ pragma solidity ^0.5.13;
 pragma experimental ABIEncoderV2;
 
 import { GovernanceScript } from "script/utils/Script.sol";
-import { console2 } from "forge-std/Script.sol";
-import { MentoUpgrade1_baklava } from "./baklava.sol";
+import { MU01_BaklavaCGP } from "./MU01-Baklava-CGP.sol";
 import { ICeloGovernance } from "mento-core/contracts/governance/interfaces/ICeloGovernance.sol";
 
 import { SwapTest } from "script/test/Swap.sol";
 import { Chain } from "script/utils/Chain.sol";
 import { Contracts } from "script/utils/Contracts.sol";
 
 // forge script {file} --rpc-url $BAKLAVA_RPC_URL
-contract MentoUpgrade1_baklava_simulation is GovernanceScript {
+contract MU01_BaklavaCGPSimulation is GovernanceScript {
```

Comment:
What do you think about adding a comment here or in our docs to let people know this isn't re-runnable yet?

Reply:
Can you expand a bit?

Reply:
I think we spoke about this before, but if someone were to run this simulation script without modifying the proposal, it would fail on initialization of the proxies.

Reply:
maybe we can mention that in the docs if not already mentioned? @bayological